com.unity.multiplayer.mlapi/Runtime/Core/MessageBatcher.cs

```diff
@@ -15,12 +16,17 @@ public class SendStream
         {
             public string channel;
             public PooledBitStream Stream = PooledBitStream.Get();
+            public PooledBitWriter Writer;
             public bool Empty = true;
+
+            public SendStream()
+            {
+                Writer = PooledBitWriter.Get(Stream);
+            }
         }
 
         // Stores the stream of batched RPC to send to each client, by ClientId
         private Dictionary<ulong, SendStream> SendDict = new Dictionary<ulong, SendStream>();
-        private PooledBitWriter Writer = new PooledBitWriter(PooledBitStream.Get());
```

Comment:
`m_***` prefixes?

Reply:
Yeop, will make those changes for the rpc batching code.